week05_explore/bayes.py

```diff
@@ -1,153 +1,81 @@
-"""
-A single-file module that makes your lasagne network into a bayesian neural net.
-Originally created by github.com/ferrine , rewritten by github.com/justheuristic for simplicity
+import torch
```

Comment:
Since this file is converted from someone's module, I believe credit should be given to the original author.